src/components/Layout/NavigationLayout/GenreNavigation/GenreNavigation.tsx

```diff
@@ -1,11 +1,51 @@
+import { GENRE_FILTER } from '../../../../constants/constants';
+import { moviesActions } from '../../../../redux/moviesSlice';
+import { useAppDispatch, useAppSelector } from '../../../../redux/store';
+
 import './GenreNavigation.scss';
 
 const GenreNavigation = (): JSX.Element => {
-  const genres = ['All', 'documentary', 'comedy', 'horror', 'crime'];
+  const filters = useAppSelector((state) => state.movies.filters);
+  const dispatch = useAppDispatch();
+  const allFilters = GENRE_FILTER[0].id;
 
-  const genresTemplate = genres.map((genre) => <div key={genre}>{genre}</div>);
+  let genres = [];
 
-  return <div className="mwGenresContainer">{genresTemplate}</div>;
+  const initMenuOptions = (): JSX.Element => {
+    GENRE_FILTER.forEach((option) => {
+      genres.push(
+        <button
+          className={`mwFilterButton ${
```

Comment:
Could you please move the whole expression (including "mwFilterButton") to a separate function, something like "getMenuButtonClassName", simply for readability?